NitroxModel/Serialization/ServerList.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
+
+namespace NitroxModel.Serialization
+{
+    public class ServerList
+    {
+        private const string SERVERS_FILE_NAME = "servers";
+        private static ServerList instance;
+        private readonly List<Entry> entries = new();
+        public static ServerList Instance => instance ??= From(DefaultFile);
+
+        private static ServerList Default
+        {
+            get
+            {
+                ServerList list = new();
+                list.Add(new Entry("local server", "127.0.0.1", 11000));
+                return list;
+            }
+        }
+
+        public static string DefaultFile => Path.Combine(NitroxAppData.Instance.LauncherPath, SERVERS_FILE_NAME);
+
+        public IEnumerable<Entry> Entries => entries;
+
+        public static ServerList From(string file = null)
+        {
+            // TODO: Remove backward compatibility after 1.5.0.0 release (no need to move old server file).
+            if (File.Exists(SERVERS_FILE_NAME))
+            {
+                File.Move(SERVERS_FILE_NAME, DefaultFile);
+            }
```

Comment:
Even if it's not a nominal scenario, we should probably go the safe way

NitroxModel/OS/FileSystem.cs
```suggestion
            // TODO: Remove backward compatibility after 1.5.0.0 release (no need to move old server file).
            FileSystem.instance.ReplaceFile(SERVERS_FILE_NAME, DefaultFile);
```

Reply:
Fixed ReplaceFile to work across drives as well.